lib/internal/module.js

```diff
@@ -50,6 +51,40 @@ function stripBOM(content) {
   return content;
 }
 
+/**
+ * Find end of shebang line and slice it off
+ */
+function stripShebang(content) {
```

Comment:
Shebang is more commonly used. For example, the [Hashbang](https://en.wikipedia.org/wiki/Shebang_(Unix)) article on Wikipedia redirects to the Shebang article.